packages/uxpin-merge-cli/src/steps/serialization/component/implementation/typescript/property/type/checker/isEnum.ts

```diff
@@ -0,0 +1,10 @@
+import * as ts from 'typescript';
+
+export function isEnum(type:ts.Type):type is ts.LiteralType {
+  if (type.symbol && type.symbol.valueDeclaration) {
+    const typeSymbol:ts.Node = type.symbol.valueDeclaration;
+    return typeSymbol.kind === ts.SyntaxKind.EnumMember;
```

Comment:
there is method ts.isEnumMember
and now we have ts 4 so you can use optional chaining so instead of writing `type.symbol && type.symbol.valueDeclaration` you can just write `type.symbol?.valueDeclaration`

Reply:
fixed. funfact: phpstorm found polaris in node modules so took ts 3 definition from dependency